drivers/peci/peci_mchp_xec.c

```diff
@@ -175,11 +170,13 @@ static int peci_xec_write(const struct device *dev, struct peci_msg *msg)
 		return -ETIMEDOUT;
 	}
 #else
-	wait_timeout = tx_buf->len;
+	/* In worst case, overall timeout will be 1msec (100 * 10usec) */
+	u8_t wait_timeout_cnt = 100;
```

Comment:
Need to update types uint8

Reply:
just fix the types everything else looks ok